environment.yml

```diff
@@ -29,6 +29,8 @@ dependencies:
     - jupyter
     - opacus
     - pillow==9
-#    - sentencepiece
-#    - gdown
-#    - pyemd==0.5.1
+    - sentencepiece
+    - gdown
+    - h5py
+    - librosa
+    - numba
```

Comment:
We may still need specific version for `numba` and `librosa`. Otherwise, it raises some errors.